packages/kotti-ui/source/kotti-style/_comments.scss

```diff
@@ -1,127 +1,128 @@
+	&__info {
+		display: flex;
+		flex-direction: row;
+		width: 100%;
+		font-size: 0.7rem;
+		line-height: 1.2rem;
+		&__name {
+			flex: 1 1;
+			font-weight: 600;
+		}
+		&__time {
+			flex: 1 1;
+			color: $lightgray-500;
+			text-align: right;
+		}
+	}
+	&__message {
+		font-size: 0.75rem;
+		line-height: 1.2rem;
+		word-break: break-word;
+	}
+	&__action {
+		display: flex;
+		flex-direction: row;
+		justify-content: space-between;
+		margin: 0.2rem 0;
+		font-size: 0.7rem;
+		font-weight: 600;
+		line-height: 1.2rem;
+		color: $lightgray-500;
+		.yoco {
+			font-size: 0.9rem;
+		}
+		&__options {
+			position: absolute;
+			top: 1.2rem;
+			right: 0;
+			z-index: $zindex-4;
+			display: none;
+			height: auto;
+			padding: 0.4rem 0;
+			line-height: 1.4rem;
+			text-align: left;
+			background: #ffffff;
+			border-radius: $border-radius;
+			box-shadow: $box-shadow;
+			li {
+				padding: 0.2rem 1rem;
+				list-style: none;
+			}
+			li:hover {
+				background: $lightgray-300;
+			}
+		}
+		&__reply {
+			display: flex;
+			flex: 0 0 auto;
+			align-items: center;
+			padding: 0 0.2rem;
+			border-radius: 0.2rem;
+			&:hover {
+				color: $darkgray-400;
+				cursor: pointer;
+				background: $lightgray-400;
+			}
+		}
+		&__more {
+			position: relative;
+			display: inline-block;
+			flex: 0 0 2rem;
+			padding: 0 0.2rem;
+			text-align: center;
+			border-radius: 0.2rem;
+			&:hover {
+				color: $darkgray-400;
+				cursor: pointer;
+				background: $lightgray-400;
+				.kt-comment__action__options {
+					display: block;
+				}
+			}
+		}
 	}
 }
 
-.comment-inline-edit {
+.kt-comment-inline-edit {
 	position: relative;
 	width: 100%;
 }
-.comment-inline-edit-buttons {
-	position: absolute;
-	right: 0;
-	z-index: $zindex-4;
-}
-
-.action__options {
+.kt-comment-inline-edit-buttons {
 	position: absolute;
-	top: 1.2rem;
 	right: 0;
 	z-index: $zindex-4;
-	display: none;
-	height: auto;
-	padding: 0.4rem 0;
-	line-height: 1.4rem;
-	text-align: left;
-	background: #ffffff;
-	border-radius: $border-radius;
-	box-shadow: $box-shadow;
-	li {
-		padding: 0.2rem 1rem;
-		list-style: none;
-	}
-	li:hover {
-		background: $lightgray-300;
-	}
-}
-
-.action__reply {
-	flex: 0 0 auto;
-	padding: 0 0.2rem;
-	border-radius: 0.2rem;
-	&:hover {
-		color: $darkgray-400;
-		cursor: pointer;
-		background: $lightgray-400;
-	}
-}
-.action__more {
-	position: relative;
-	display: inline-block;
-	flex: 0 0 2rem;
-	padding: 0 0.2rem;
-	text-align: center;
-	border-radius: 0.2rem;
-	&:hover {
-		color: $darkgray-400;
-		cursor: pointer;
-		background: $lightgray-400;
-		.action__options {
-			display: block;
-		}
-	}
 }
 
 // Comment Reply => KtCommentReply
 
-.comment-reply {
+.kt-comment-reply {
 	display: flex;
 	flex-direction: row;
 	padding: 0.2rem 0;
+	&__action {
+		display: flex;
+		flex-direction: row;
+		justify-content: space-between;
+		margin: 0.2rem 0;
+		font-size: 0.7rem;
+		font-weight: 600;
+		line-height: 1.2rem;
+		color: $lightgray-500;
```

Comment:
can u move this to the css var notation

we have text color tokens now, you should pick the closest color that fits you from https://3yourmind.github.io/kotti/foundations/colors/list-of-tokens

might be `text-02` or `text-03`

Reply:
You should actually consult the designs. Linked in the docs as always